pipe/tchain/tchain_date_gap_filler.yaml

```diff
@@ -2,9 +2,9 @@
 pipeline:
   name: tchain_date_gap_filler
 transform:
-  image_pull_secrets:
-  - battelleecology-quay-read-all-pull-secret
-  image: quay.io/battelleecology/date_gap_filler:c0ce8112
+  # image_pull_secrets:
+  # - battelleecology-quay-read-all-pull-secret
+  image: us-central1-docker.pkg.dev/neon-shared-service/neonscience/neon-is-date-gap-fill:v03.01.01
```

Comment:
Please use single digits for semantic version. If there is no existing semantic version in the existing quay.io repo, please start with v1.0.0